src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -78,9 +107,19 @@ private DataSourceUtils() {}
         // Go through our directories:
         final Set<String> names = new LinkedHashSet<>();
         for ( final String pathString : dataSourceDirectories ) {
+
+            logger.info("Initializing data sources from directory: " + pathString);
+
             final Path p = IOUtils.getPath(pathString);
             if ( !isValidDirectory(p) ) {
-                logger.warn("WARNING: Given path is not a valid directory: " + p.toUri().toString());
+                logger.error("ERROR: Given data source path is not a valid directory: " + p.toUri().toString());
```

Comment:
I notice that you have a lot of error messages that do not throw exceptions.  Is this correct behavior?

Reply:
It's an error to have an invalid data source path.

You're right - I'll throw an exception here.